### PR TITLE
feat: chat timestamps

### LINKED ISSUE #186: Add timestamps to chat messages
It'll be helpful to have timestamps for chat messages

### PR DESCRIPTION
**Pull request by nadirabbas**
Fixes #186

Gets UTC from server-side and converts to local.

image

### CODE REVIEW

File: kofta/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "@types/jest": "^26.0.15",
     "@types/node": "^12.0.0",
     "@types/react-dom": "^16.9.8",
+    "dateformat": "^4.5.1",
```
Comment (benawad) [inline kofta/package.json]: can you use date-fns so we use the same date lib for everything

Comment (nadirabbas) [reply kofta/package.json]: Okay sure.

File: kofta/src/vscode-webview/utils/format.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var { format: dfsFormat } = require("date-fns");
```
Comment (benawad) [inline kofta/src/vscode-webview/utils/format.ts]: use import syntax

**Comment by benawad** [comment]
conflicts

**Comment by nadirabbas** [comment]
oh they weren't there a minute ago lol

**Pull request merged**